src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java

```diff
@@ -14,8 +13,10 @@
 
 public class ChatTests extends NewTestTemplate {
 
-  private String MESSAGE_ON_MAIN_CHAT = "Test message on main chat";
-  private String MESSAGE_ON_PRIVATE_CHAT = "Test message on private chat";
+  private final static String USER_IN_PRIVATE_SECTION_NOT_DISPLAYED_ERROR = "USER IS NOT DISPLAYED IN PRIVATE SECTION";
+  private final static String MESSAGE_ON_MAIN_CHAT = "Test message on main chat";
+  private final static String MESSAGE_ON_PRIVATE_CHAT = "Test message on private chat";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AModifiersOrderCheck)